dtschema/schemas/options/u-boot.yaml

```diff
@@ -71,6 +71,21 @@ properties:
       2: use simplified command line (e.g. avoid hush)
       3... reserved
 
+  bootscr-address:
+    $ref: /schemas/types.yaml#/definitions/uint64
+    default: 0x0
+    description:
+      This property holds the full address of the boot script file. This helps in
+      making automated flows easier by fetching address directly from DT.
+
+  bootscr-ram-offset:
+    $ref: /schemas/types.yaml#/definitions/uint64
+    default: 0x0
+    description:
+      This property holds the boot script file offset from start of the ram base
```

Comment:
Again you shouldn't say 'This property' since we know it is a property :-)

Reply:
also again please mention where this address ends up and what it affects. The bindings just need more detail so that people can understand it a bit better (what it is for, how it works, why I might use it, etc.)